insights/parsers/cpuset_cpus.py

```diff
@@ -1,13 +1,14 @@
 """
-CpuSetCpus - File ``/sys/fs/cgroup/cpuset/cpuset.cpus``
-=======================================================
+cpuset_cpus - File ``/sys/fs/cgroup/cpuset/cpuset.cpus``
+========================================================
 
 This parser reads the content of ``/sys/fs/cgroup/cpuset/cpuset.cpus``.
 This file shows the default cgroup cpuset.cpu of system. The format
 of the content is string including comma.
 """
 
-from .. import parser, CommandParser
+from insights import ContainerParser
+from insights import parser, CommandParser
```

Comment:
Could you change these to full path imports please?

```
from insights.core import CommandParser, ContainerParser
from insights.core.plugins import parser
```

Reply:
@ryan-blakley - Updated, please check

Reply:
Thank you.